packages/storybook/stories/CustomIcons/utils.tsx

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { mergeStyles, Stack, TextField } from '@fluentui/react';
+import React, { useState } from 'react';
+
+const iconGrid = (allIcons: Record<string, JSX.Element>, searchText?: string): JSX.Element[] => {
+  const allIconKeys = Object.keys(allIcons);
+  const filteredKeys = searchText
+    ? allIconKeys.filter((k) => k.toLowerCase().includes(searchText.toLowerCase()))
+    : allIconKeys;
+
+  return filteredKeys.map((key, idx) => {
+    return (
+      <Stack
+        key={idx}
+        horizontal
+        style={{
+          padding: '0.5rem 0'
+        }}
+      >
+        <Stack horizontalAlign="start" className={mergeStyles({ span: { fontSize: '1rem' } })}>
```

Comment:
name and icon aren't quite centered vertically here:
![image](https://user-images.githubusercontent.com/2684369/143654803-b8b362f8-e53b-40d8-9678-81e004e8caff.png)

For beta banner I had the same issue, this worked for me:
![image](https://user-images.githubusercontent.com/2684369/143655128-9a83ec9f-91b5-4be6-8e00-3f541a56c1b9.png)

Reply:
Fixed. A simple fix is to wrap everything in Stack with verticalAlign center and give a line height of 0 to the icon ;)